cmd/clusterctl/client/alpha/rollout.go

```diff
@@ -30,6 +30,7 @@ type Rollout interface {
 	ObjectRestarter(cluster.Proxy, util.ResourceTuple, string) error
 	ObjectPauser(cluster.Proxy, util.ResourceTuple, string) error
 	ObjectResumer(cluster.Proxy, util.ResourceTuple, string) error
+	ObjectRollbacker(cluster.Proxy, util.ResourceTuple, string, int64) error
```

Comment:
is it required to use int64?

Reply:
Yes, the underlying `mdutil.Revision()` func returns a int64.